distributed/scheduler.py

```diff
@@ -7543,10 +7550,12 @@ def adaptive_target(self, target_duration=None):
         # CPU
 
         # TODO consider any user-specified default task durations for queued tasks
-        queued_occupancy = len(self.queued) * self.UNKNOWN_TASK_DURATION
+        queued_occupancy: float = len(self.queued) * self.UNKNOWN_TASK_DURATION
+        # TODO: threads per worker
+        # TODO don't include network occupancy?
```

Comment:
Interesting question (way out of scope): Should high network occupancy maybe even act as a suppressing factor?